bin.src/ingestPredefinedFluxCal.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import typing
+
+from lsst.daf.butler import Butler, DatasetRef, DatasetType, DimensionGroup, FileDataset
+from lsst.resources import ResourcePath
+
+
+def main() -> None:
+    """Ingest a predefined fluxCal.
+
+    This program registers a single fluxCal many times
+    with various visit numbers
+    so that the pipeline will use it to calibrate all these visits.
+    """
+    parser = argparse.ArgumentParser(
+        description=typing.cast(str, main.__doc__),
+    )
+    parser.add_argument(
+        "fluxCal",
+        help="""
+        Path to a predefined fluxCal
+        """,
+    )
+    parser.add_argument(
+        "-b",
+        "--butler-config",
+        metavar="URI",
+        required=True,
+        help="""
+        [Required] URI for datastore repository
+        """,
+    )
+    parser.add_argument(
+        "-d",
+        "--data-query",
+        metavar="SQL",
+        required=True,
+        help="""
+        [Required] Query string to choose visits
+        to which to apply the predefined fluxCal.
+        """,
+    )
+    parser.add_argument(
+        "-i",
+        "--input",
+        metavar="COLLECTION[,...]",
+        required=True,
+        help="""
+        [Required] Comma-separated names of the collection(s)
+        to search for pfsConfig files.
+        """,
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        metavar="RUN",
+        required=True,
+        help="""
+        [Required] Run collection to which to register the specified fluxCal.
+        """,
+    )
+    parser.add_argument(
+        "--transfer",
+        default="auto",
+        choices=("auto", "move", "copy", "direct", "split", "link", "hardlink", "relsymlink", "symlink"),
+        help="""
+        Mode for transferring files into the datastore
+        """,
+    )
+    args = parser.parse_args()
+
+    input_collections = [x.strip() for x in args.input.split(",")]
+    input_collections = [x for x in input_collections if x]
```

Comment:
Is it necessary to process the `args.input`? I think the butler does all this for you, doesn't it?

Reply:
According to my experiment, comma-separated collection names in a single string was not recognized by `Butler`.
```
>>> butler=Butler.from_config(config=".", collections="test,test2,test3")
>>> butler.query_datasets("fluxCal", find_first=False)
lsst.daf.butler._exceptions.MissingCollectionError: No collection with name 'test,test2,test3' found.
```